### ISSUE TITLE
js-yaml advisory

### ISSUE THREAD

**Issue by dekaikiwi** [boopathi/react-svg-loader#254]
Is there any plan to update the version of `js-yaml` at all?

Currently `yarn audit` yields the following advisory.

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ moderate      │ Denial of Service                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ js-yaml                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=3.13.0                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ react-svg-loader                                             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ react-svg-loader > react-svg-core > svgo > js-yaml           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/788                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

**Comment by lasseborly** [comment]
It seems to already be fixed in this commit: https://github.com/boopathi/react-svg-loader/commit/e8884d1c1aea1d54e406479b795099b0e74aab75

Would be ice to have a version out with the fix :)

**Comment by cakasuma** [comment]
when will be the next release to fix this issue ?

**Comment by deregtd** [comment]
Adding another request here for a release push!

**Comment by james-willett** [comment]
Would be great if this could be pushed 😄

**Comment by michaelnyu** [comment]
An additional request for a release push 💃  😄

**Comment by boopathi** [comment]
Published 3.0.0
